lib/maverick/arguments.ex

```diff
@@ -0,0 +1,21 @@
+defmodule Maverick.Arguments do
+  def validate(args) do
+    args
+  end
+
+  def get(%Maverick.Route{} = route, %Plug.Conn{} = conn) do
+    Enum.map(route.args, &arg(conn, &1))
+  end
+
+  defp arg(conn, :params), do: conn.params
+
+  defp arg(conn, :conn), do: conn
+
+  defp arg(conn, {parameter, :required}) do
+    get_in(conn.params, List.wrap(parameter))
```

Comment:
this is the same as the non-required version below; should this blow up if the `get_in` fails?

Reply:
yeah, I forgot I had this in there, I will pull it out of this PR and deal with required params later.